src/test/resources/test_data/overall_calculation/hdc4/crs-2135-hdced-tranche1-bug.json

```diff
@@ -0,0 +1,138 @@
+{
+  "offender":{
+    "reference":"ABC123D",
+    "dateOfBirth":"2000-07-23",
+    "isActiveSexOffender":false
+  },
+  "bookingId":2860569,
+  "sentences":[
+    {
+      "type":"StandardSentence",
+      "offence":{
+        "committedAt":"2020-07-31",
+        "offenceCode":"MD71134C"
+      },
+      "duration":{
+        "durationElements":{
+          "DAYS":0,
+          "WEEKS":0,
+          "YEARS":0,
+          "MONTHS":24
+        }
+      },
+      "isSDSPlus":false,
+      "identifier":"2053534c-f153-3be5-90e6-efdee507e96d",
+      "recallType":"STANDARD_RECALL",
+      "sentencedAt":"2023-10-27",
+      "caseSequence":3,
+      "lineSequence":2,
+      "caseReference":"recall",
+      "consecutiveSentenceUUIDs":[
+      ],
+      "hasAnSDSEarlyReleaseExclusion":"NO"
+    },
+    {
+      "type":"StandardSentence",
+      "offence":{
+        "committedAt":"2020-07-31",
+        "offenceCode":"MD71131C"
+      },
+      "duration":{
+        "durationElements":{
+          "DAYS":0,
+          "WEEKS":0,
+          "YEARS":0,
+          "MONTHS":24
+        }
+      },
+      "isSDSPlus":false,
+      "identifier":"2053534c-f153-3be5-90e6-efdee507e96d",
+      "recallType":"STANDARD_RECALL",
+      "sentencedAt":"2023-10-27",
+      "caseSequence":3,
+      "lineSequence":2,
+      "caseReference":"recall",
+      "consecutiveSentenceUUIDs":[
+      ],
+      "hasAnSDSEarlyReleaseExclusion":"NO"
+    },
+    {
+      "type":"StandardSentence",
+      "offence":{
+        "committedAt":"2023-12-15",
+        "offenceCode":"SC15005"
+      },
+      "duration":{
+        "durationElements":{
+          "DAYS":0,
+          "WEEKS":0,
+          "YEARS":0,
+          "MONTHS":19
+        }
+      },
+      "isSDSPlus":false,
+      "identifier":"a19c0711-24d0-3ad5-bf87-8b42eafe306d",
+      "recallType":null,
+      "sentencedAt":"2024-04-09",
+      "caseSequence":4,
+      "lineSequence":3,
+      "caseReference":"01PY1388023",
```

Comment:
These case references should really be removed so that the data is anonomous. I have created a helper test for this `BookingAnonmiser`